helpdesk_mgmt/i18n/helpdesk_mgmt.pot

```diff
@@ -1220,3 +1220,37 @@ msgstr ""
 msgid "Website communication history"
 msgstr ""
 
+#. module: helpdesk_mgmt
```

Comment:
Please don't directly edit  .pot file, as the file is generated by weblate and this change could produce conflicts

Reply:
Sorry. That is not the first time i forgot that.